pkg/network/setup/network_test.go

```diff
@@ -373,7 +375,8 @@ var _ = Describe("VMNetworkConfigurator", func() {
 		var configState configStateUnplugger
 
 		BeforeEach(func() {
-			vmi = api2.NewMinimalVMIWithNS("testnamespace", "testVmName")
+			vmi = &v1.VirtualMachineInstance{ObjectMeta: metav1.ObjectMeta{UID: "123"}}
+			vmi.Spec.Networks = []v1.Network{}
```

Comment:
Are these leftovers?

Reply:
What do you mean by "these"? I don't see any leftovers in the test. The empty network list is needed to avoid NPE.